plugins/git/functions

```diff
@@ -33,7 +33,15 @@ git_build_app_repo() {
   local TMP_TAG="dokku/$REV"
   chmod 755 "$GIT_BUILD_APP_REPO_TMP_WORK_DIR"
   unset GIT_DIR GIT_WORK_TREE
-  [[ ! -d "$DOKKU_ROOT/$APP" ]] && apps_create "$APP"
+  local DOKKU_GLOBAL_DISABLE_AUTOCREATE="$(config_get --global DOKKU_DISABLE_APP_AUTOCREATION)"
+  if [[ ! -d "$DOKKU_ROOT/$APP" ]]; then
+    if [[ "$DOKKU_GLOBAL_DISABLE_AUTOCREATE" == true ]]; then
+      echo $'\e[1G\e[K'"-----> ERROR: Invalid app name '$APP' specified."
```

Comment:
Can you use the `dokku_log_warn` helpers here instead?

Reply:
I think I'll use `dokku_log_fail`, since this isn't a recoverable error will exit properly. Thanks!